ansible/roles/ceph-grafana/tasks/configure_grafana.yml

```diff
@@ -1,10 +1,45 @@
 ---
-- name: Start Grafana
+- name: Make sure grafana is down
+  service:
+    name: grafana-server
+    state: stopped
+
+- name: Wait for grafana to be stopped
+  wait_for:
+    port: 3000
+    state: stopped
+
+- set_fact:
```

Comment:
Yeah, github seems to be a bit slow today but the updated patch will hopefully show up soon.